src/main/scala/higherkindness/rules_scala/common/sbt-testing/Test.scala

```diff
@@ -29,10 +29,21 @@ object TestHelper {
     f: Runner => A
   ) = {
     val options =
-      if (framework.name == "specs2") {
-        // Get rid of any extra context before the test name (specs2 doesn't recognize it)
-        val testName = scopeAndTestName.replaceAll(raw"(?<=\\Q).*::", "")
-        Array("-ex", testName)
+      if (framework.name == "specs2" && scopeAndTestName.contains("::")) {
+        // scopeAndTestName from intellij has the format '\Q${scope}::${test}\E$' for individual tests,
+        // and '\Q${scope}\E::' for scopes (see getTestFilter in com.google.idea.blaze.scala.run.Specs2Utils).
+        // specs2 filters only on test, not on scope. If we get just a scope, pass nothing.
```

Comment:
Pardon my ignorance here, by passing nothing do we do no filtering?

Asked another way, is there a situation in which the user in IntelliJ ends up running more tests than they they thought they would?

Reply:
That is correct. If someone hits the run button by a scope, it will run all the tests in that file. That was the existing behavior prior to my previous PR.

The intent of my previous PR was:
* to change the functionality of `bazel test ... --test_filter="my.test.Class#\Qfoo::bar\E"` so that special regex characters in `bar` would be interpreted literally instead of as special regex characters

The actual effect of my previous PR was:
* `bazel test ... --test_filter="my.test.Class#\Qfoo::bar\E"` interprets special regex characters in `bar` literally
* `bazel test ... --test_filter="my.test.Class#\Qfoo\E::"`, which used to run all tests in `my.test.Class`, now runs none of them

This PR undoes the second change, which was not intentional. I.e. it is changing the behavior opposite a change in behavior I accidentally included in my previous PR